musl >= 1.1.22 uses the membarrier(2) syscall, allow it in the sandbox.

See: https://bugs.alpinelinux.org/issues/10257

diff -upr chromium-73.0.3683.103.orig/sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc
--- sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc.orig	2019-04-26 10:56:56.824506143 +0200
+++ sandbox/linux/seccomp-bpf-helpers/syscall_sets.cc	2019-04-26 11:03:14.060482592 +0200
@@ -386,6 +386,7 @@ bool SyscallSets::IsAllowedProcessStartO
   switch (sysno) {
     case __NR_exit:
     case __NR_exit_group:
+    case __NR_membarrier:
     case __NR_wait4:
     case __NR_waitid:
 #if defined(__i386__)
